src/index.html

```diff
@@ -1,13 +1,40 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <section class="card" data-qa="card">
+      <img src="images/imac.jpeg" alt="Apple-imack"
+      class="card__image"
+      width="160" height="134">
```

Comment:
Pay attention on code style
```suggestion
      <img
        src="images/imac.jpeg"
         alt="Apple-imack"  
         class="card__image"      
         width="160" height="134"
       >
```